core/commonTest/src/contract/map/ImmutableMapTest.kt

```diff
@@ -283,13 +283,16 @@ abstract class ImmutableMapTest {
     }
 
     @Test fun noOperation() {
-        immutableMapOf<String, String>().toPersistentMap().testNoOperation({ clear() }, { clear() })
+        immutableMapOf<Any, String>().toPersistentMap().testNoOperation({ clear() }, { clear() })
```

Comment:
Perhaps this change of the empty map types was not needed to fix the test

Reply:
Yep, reverted